package.json

```diff
@@ -37,32 +37,32 @@
     "test:e2e:firefox:grid": "mocha --firefox --grid",
     "test:e2e:opera": "mocha --opera",
     "test:e2e:safari": "mocha --safari",
-    "test:server:start": "http-server &",
+    "test:server:start": "http-server -p 8080 &",
     "release": "np",
     "release:prepare": "npm explore vl-ui-util -- npm run release-component:prepare $(pwd) datepicker",
     "release:testless": "np --yolo",
     "demo": "http-server"
   },
   "dependencies": {
-    "vl-ui-button": "^4.0.0",
-    "vl-ui-core": "^5.0.1",
-    "vl-ui-icon": "^4.0.0",
+    "vl-ui-button": "^5.0.1",
+    "vl-ui-core": "^5.0.3",
+    "vl-ui-icon": "^4.0.1",
     "vl-ui-input-field": "^3.0.0",
     "vl-ui-input-group": "^4.0.0",
-    "vl-ui-select": "^3.0.2"
+    "vl-ui-select": "^4.0.1"
   },
   "peerDependencies": {
-    "vl-ui-button": "^4.0.0",
-    "vl-ui-core": "^5.0.1",
-    "vl-ui-icon": "^4.0.0",
+    "vl-ui-button": "^5.0.1",
+    "vl-ui-core": "^5.0.3",
+    "vl-ui-icon": "^4.0.1",
     "vl-ui-input-field": "^3.0.0",
     "vl-ui-input-group": "^4.0.0"
   },
   "devDependencies": {
     "@govflanders/vl-ui-datepicker": "3.9.1",
-    "vl-ui-select": "^3.0.2",
+    "vl-ui-select": "^4.0.1",
```

Comment:
rare dependency om hier te hebben, maar dat is wschl weeral omdat de 2 niet goed samen spelen?

Reply:
of is dat om een dropdown te hebben? we gingen toch iets Select-achtigs in vl-core steken, nee?